shared/chat/inbox/row/big-team-header/index.js

```diff
@@ -27,19 +27,15 @@ class BigTeamHeader extends React.PureComponent<Props> {
         <Text type="BodySmallSemibold" style={teamStyle}>
           {props.teamname}
         </Text>
-        <Icon
-          className="icon"
-          type="iconfont-gear"
+        <ClickableBox
           onClick={isMobile ? () => props.onClickGear() : props.onClickGear}
```

Comment:
Why is this necessary?

Reply:
Just so that taps that hit the blue badge box rather than the gear icon also fire the onclick.  Make sense?

Reply:
But both call the same props and it's an arrow closure so the second case just works right

Reply:
Ah, thought you were asking about the ClickableBox.  Surprisingly, it's necessary:
```js
const mapDispatchToProps = (dispatch, {teamname}) => ({
  onClickGear: (evt?: SyntheticEvent<Element>) =>
    dispatch(
      navigateAppend([
        {
          props: {
            teamname,
            isSmallTeam: false,
            position: 'bottom right',
            targetRect: evt && evt.currentTarget.getBoundingClientRect(),
          },
          selected: 'infoPanelMenu',
        },
      ])
    ),
})
```
So, we'll have an `evt` passed through by default, and on mobile we want to suppress that so that we don't try to call `getBoundingClientRect()` on it, which doesn't exist on mobile.

Seems like it'd be preferable to instead just test for `getBoundingClientRect`'s existence inside the dispatch func and call it if it's there?  CC @buoyad, this is relative popup stuff.